Formula/cloudstate-cli.rb

```diff
@@ -0,0 +1,14 @@
+class CloudstateCli < Formula
+  desc "Cloudstate CLI"
+  homepage "https://github.com/sleipnir/cloudstate-cli"
+  url "https://github.com/sleipnir/cloudstate-cli/releases/download/0.3.54/cloudstate-0.3.54-osx.tar.gz"
```

Comment:
Please don't use binary downloads. All formula must build from source: https://docs.brew.sh/Acceptable-Formulae#we-dont-like-binary-formulae

Reply:
Ok. I will try to create a cask

Reply:
Any chance of making an exception? This app is built on top of Rust and would have to depend on Cargo and rustup to be compiled directly.

Reply:
Yes, a lot of formula do.

Reply:
Cool. Any example ?